lib/nexpose/scan_template.rb

```diff
@@ -491,5 +491,39 @@ def self.copy(nsc, id)
     def delete(nsc)
       nsc.delete_scan_template(id)
     end
+
+    # Enable or disable asset configuration scanning for this template.
+    #
+    # @param [String] "full" to enable asset configuration logging, and
+    #   "default" or "none" to disable it.
+    def aces_level=(level)
+      logging = REXML::XPath.first(@xml, 'ScanTemplate/Logging')
+      if (logging.nil?)
+        logging = REXML::Element.new('Logging')
+        @xml.add_element(logging)
+      end
+      aces = REXML::XPath.first(logging, 'aces')
+      if (aces.nil?)
+         aces = REXML::Element.new('aces')
+         logging.add_element(aces)
+      end
+      aces.attributes['level'] = level
+    end
+
+    # @return [String] the asset configuration logging value for this
+    #   template.
+    def aces_level
+      logging = REXML::XPath.first(@xml, 'ScanTemplate/Logging')
+      return 'default' if logging.nil?
+      aces = REXML::XPath.first(logging, 'aces')
+      return 'default' if aces.nil?
+      return aces.attributes['level']
```

Comment:
Minor, but this could return `nil` here if `aces_level=` is called with `nil`.  Perhaps only valid values should be allowed as args to `aces_level` and/or an attribute existence test is in order here.